config/locales/en.yml

```
@@ -2357,6 +2358,8 @@ en:
         description: "Share work packages with users who are not members of the project."
       project_list_sharing:
         description: "Share project lists with individual users."
+      calculated_values:
+        description: "Calculated values allow you to create a mathematical formula based attribute using numeric values and other project attributes and custom fields."
```

Spotted another one here:

```suggestion
        description: "Calculated Values allow you to create a mathematical formula based attribute using numeric values and other project attributes and custom fields."
```

---

Forgot to change it too